package.json

```diff
@@ -1,87 +1,87 @@
-    "preprocessor",
-    "sass",
-    "scss",
-    "style"
-  ],
-  "dependencies": {
-    "async-foreach": "^0.1.3",
-    "chalk": "^1.1.1",
-    "cross-spawn": "^3.0.0",
-    "gaze": "^1.0.0",
-    "get-stdin": "^4.0.1",
-    "glob": "^7.0.3",
-    "in-publish": "^2.0.0",
-    "lodash": "^4.17.11",
-    "meow": "^3.7.0",
-    "mkdirp": "^0.5.1",
-    "nan": "^2.13.2",
-    "node-gyp": "^3.8.0",
-    "npmlog": "^4.0.0",
-    "request": "^2.88.0",
-    "sass-graph": "^2.2.4",
-    "stdout-stream": "^1.4.0",
-    "true-case-path": "^1.0.2"
-  },
-  "devDependencies": {
-    "coveralls": "^3.0.2",
-    "eslint": "^3.4.0",
-    "fs-extra": "^0.30.0",
-    "istanbul": "^0.4.2",
-    "mocha": "^3.1.2",
-    "mocha-lcov-reporter": "^1.2.0",
-    "object-merge": "^2.5.1",
-    "read-yaml": "^1.0.0",
-    "rimraf": "^2.5.2",
-    "sass-spec": "https://github.com/sass/sass-spec.git#dc2d573",
-    "unique-temp-dir": "^1.0.0"
-  }
-}
+	"name": "node-sass",
+	"version": "4.12.1",
+	"libsass": "3.6.1-28-g2b8a",
+	"description": "Wrapper around libsass",
+	"license": "MIT",
+	"bugs": "https://github.com/sass/node-sass/issues",
+	"homepage": "https://github.com/sass/node-sass",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/sass/node-sass"
+	},
+	"author": {
+		"name": "Andrew Nesbitt",
+		"email": "andrewnez@gmail.com",
+		"url": "http://andrew.github.com"
+	},
+	"engines": {
+		"node": ">=0.10.0"
+	},
+	"main": "lib/index.js",
+	"nodeSassConfig": {
+		"binarySite": "https://github.com/sass/node-sass/releases/download"
+	},
+	"bin": {
+		"node-sass": "bin/node-sass"
+	},
+	"gypfile": true,
+	"scripts": {
+		"coverage": "node scripts/coverage.js",
+		"install": "node scripts/install.js",
+		"postinstall": "node scripts/build.js",
+		"lint": "node_modules/.bin/eslint bin/node-sass lib scripts test",
+		"test": "node_modules/.bin/mocha test/{*,**/**}.js",
+		"build": "node scripts/build.js --force",
+		"prepublish": "not-in-install && node scripts/prepublish.js || in-install"
+	},
+	"files": [
+		"bin",
+		"binding.gyp",
+		"lib",
+		"scripts",
+		"src",
+		"test",
+		"vendor"
+	],
+	"keywords": [
+		"css",
+		"libsass",
+		"preprocessor",
+		"sass",
+		"scss",
+		"style"
+	],
+	"dependencies": {
+		"async-foreach": "^0.1.3",
+		"chalk": "^1.1.1",
+		"cross-spawn": "^3.0.0",
+		"gaze": "^1.0.0",
+		"get-stdin": "^4.0.1",
+		"glob": "^7.0.3",
+		"in-publish": "^2.0.0",
+		"lodash": "^4.17.11",
+		"meow": "^3.7.0",
+		"mkdirp": "^0.5.1",
+		"nan": "^2.13.2",
+		"node-gyp": "^3.8.0",
+		"npmlog": "^4.0.0",
+		"request": "^2.88.0",
+		"sass-graph": "^2.2.4",
+		"stdout-stream": "^1.4.0",
+		"true-case-path": "^1.0.2"
+	},
+	"devDependencies": {
+		"coveralls": "^3.0.2",
+		"eslint": "^3.4.0",
+		"fs-extra": "^0.30.0",
+		"istanbul": "^0.4.2",
+		"mocha": "^3.1.2",
+		"mocha-lcov-reporter": "^1.2.0",
+		"object-merge": "^2.5.1",
+		"read-yaml": "^1.0.0",
+		"rimraf": "^2.5.2",
+		"sass-spec": "https://github.com/sass/sass-spec.git#dc2d573",
```

Comment:
There seems to be a bit too much whitespace change, cna you check?

You need to figure out the right commit from the https://github.com/sass/sass-spec repository, it will be probably close to the master branch.